src/vs/workbench/contrib/interactiveEditor/browser/interactiveEditorWidget.ts

```diff
@@ -461,8 +463,28 @@ export class InteractiveEditorWidget {
 		this._previewDiffEditor.getOriginalEditor().setHiddenAreas(hiddenOriginal.map(lineRangeAsRange), 'diff-hidden');
 		this._previewDiffEditor.getModifiedEditor().setHiddenAreas(hiddenModified.map(lineRangeAsRange), 'diff-hidden');
 		this._previewDiffEditor.revealLine(modifiedLineRange.startLineNumber, ScrollType.Immediate);
-
 		this._onDidChangeHeight.fire();
+		if (this._accessibilityService.isScreenReaderOptimized()) {
+			this._previewDiffEditor.onDidUpdateDiff(() => {
+				const modified = this._previewDiffEditor.getModifiedEditor().getDomNode();
+				const original = this._previewDiffEditor.getOriginalEditor().getDomNode();
+				const container = this._previewDiffEditor.getContainerDomNode();
+				if (!modified || !original) {
+					return;
+				}
+				// prevent escape from focusing the underlying editors
+				this._previewDiffEditor.getContainerDomNode().classList.add('screen-reader-interactive-editor-widget');
+				// prevent the user from having to tab through the underlying editors
+				modified.querySelector('.previewDiff textarea.inputarea')?.setAttribute('disabled', '');
+				original.querySelector('.previewDiff textarea.inputarea')?.setAttribute('disabled', '');
+				// open the diff review
```

Comment:
Reaching into the internals of the widget seems wrong. I don't know if there is real API for this but we cannot do things like, it's too fragile.

Reply:
Yep I couldn't find real API for this but agree this is a hack